test/integration/patients/worklist/schedule.js

```diff
@@ -897,6 +899,123 @@ context('schedule page', function() {
 
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .get('.sidebar__heading')
+      .should('contain', 'States');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .find('.fa-square-check')
+      .should('have.length', 2);
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .should('contain', 'To Do')
+      .should('contain', 'In Progress');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .find('[data-check-region]')
+      .first()
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem('schedule_11111-v4'));
+
+        expect(storage.filters.states).to.deep.equal(['33333']);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[state]=33333')
+      .should('not.contain', 'filter[state]=22222');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-states-filters-region]')
+      .find('[data-check-region]')
+      .eq(1)
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem('schedule_11111-v4'));
+
+        expect(storage.filters.states).to.deep.equal([]);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', `filter[state]=${ NIL_UUID }`);
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '1');
+
+    cy
+      .get('@filtersSidebar')
+      .find('[data-filter-button]')
+      .first()
+      .click();
+
+    cy
+      .get('.picklist__item')
+      .contains('Group One')
+      .click()
+      .wait('@routeActions');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .should('contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.js-clear-filters')
+      .click()
+      .then(() => {
+        const storage = JSON.parse(localStorage.getItem('schedule_11111-v4'));
+
+        expect(storage.filters.states).to.deep.equal(['22222', '33333']);
+      })
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[state]=22222,33333');
+
+    cy
+      .get('.list-page__filters')
+      .find('[data-all-filters-region]')
+      .should('not.contain', '2');
+
+    cy
+      .get('@filtersSidebar')
+      .find('.sidebar__heading')
+      .should('not.contain', '2');
+
```

Comment:
Same as the worklist test additions, minus the `Shared By` and `Done < 30 Days` list testing.